src/electionguard-ui/ElectionGuard.UI.Lib/Models/PlaintextSpoiledBallotRecord.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace ElectionGuard.UI.Lib.Models;
+
+public partial class PlaintextSpoiledBallotRecord : DatabaseRecord
+{
+    [ObservableProperty]
+    private string _tallyId = string.Empty;
+
+    [ObservableProperty]
+    private string? _plaintextBallotData;

```

Comment:
Why do we want this nullable?

Reply:
Since the constructor was not setting the value, the linter wanted it to either be set by the constructor or made nullable.